Makefile

```diff
@@ -24,21 +32,24 @@ force:
 
 bin/calicoctl: vendor $(GO_FILES) glide.*
 	mkdir -p bin
-	go build -o "$@" "./calicoctl/calicoctl.go"
+	go build -o "$@" $(LDFLAGS) "./calicoctl/calicoctl.go"
 
-release/calicoctl: vendor force
+release/calicoctl: build-containerized
 	mkdir -p release
-	cd build-calicoctl && docker build -t calicoctl-build .
-	docker run --rm -v `pwd`:/libcalico-go calicoctl-build /libcalico-go/build-calicoctl/build.sh
+	mv bin/calicoctl release/calicoctl
+	rm -rf bin
+	mv release/calicoctl release/calicoctl-$(CALICOCTL_VERSION)
+	cd release && ln -sf calicoctl-$(CALICOCTL_VERSION) calicoctl
 
 # Build calicoctl in a container.
-build-containerized: $(BUILD_CONTAINER_MARKER)
-	mkdir -p dist
-	docker run -ti --rm --privileged --net=host \
-	-e PLUGIN=calico \
+build-containerized: force
+	mkdir -p bin
+	docker build -t calicoctl-build .
```

Comment:
I am right now reusing build-containerized in release/calicoctl, and when building for release/calicoctl I wanted a clean build environment which led me to recreating the container all the time. While writing this I just realized that the built container has just basic dependencies and nothing more and I can reuse the built container. I'll re-add BUILD_CONTAINER_MARKER.